{
  "FIND_CRM_CUSTOMER_BY_ID_SUCCESS": {
    "en": "CRM customer data was successfully extracted by %s id",
    "ru": "Данные CRM клиента были успешно извлечены по %s идентификатору",
    "az": "CRM müştərinin məlumatları %s identifikator üzrə uğurla çıxarıldı"
  },
  "FIND_CRM_CUSTOMER_BY_NAME_SUCCESS": {
    "en": "CRM customer data was successfully extracted by %s name",
    "ru": "Данные Crm клиента были успешно извлечены по %s имени",
    "az": "CRM müştərinin məlumatları uğurla çıxarıldı %s ad üzrə"
  },
  "CRM_CUSTOMER_SAVED_SUCCESSFULLY": {
    "en": "CRM customer was successfully saved",
    "ru": "Клиент Crm был успешно сохранен",
    "az": "CRM müştərisi uğurla yadda saxlanıldı"
  },
  "CRM_CUSTOMER_UPDATED_SUCCESSFULLY": {
    "en": "CRM customer was successfully updated",
    "ru": "Клиент CRM был успешно обновлен",
    "az": "CRM müştərisi uğurla yeniləndi"
  },
  "CRM_CUSTOMER_EXCEPTION_FAILED_UPDATE": {
    "en": "CRM Mongo repository has thrown exception during document update : %s",
    "ru": "CRM Монго репозиторий выдал исключение во время обновления документа : %s",
    "az": "CRM Mongo repositoriyi sənəd yeniləməsi zamanı texniki xəta gaytarıb : %s"
  },
  "CRM_CUSTOMER_EXCEPTION_FAILED_SAVE": {
    "en": "CRM Mongo repository has thrown exception during document save : %s",
    "ru": "CRM Монго репозиторий выдал исключение во время сохранения документа : %s",
    "az": "CRM Mongo repositoriyi sənəd yadda saxlanılması zamanı texniki xəta gaytarıb : %s"
  },
  "CRM_CUSTOMER_MONGO_DOCUMENT_BY_ID_EXCEPTION": {
    "en": "CRM customers mongo repository has thrown exception during document by id extraction : %s",
    "ru": "В репозитории клиентов CRM, во время извлечения по идентификатору произошла ошибка : %s",
    "az": "CRM müştərilər repositoriyda, identifikator üzrə mətumatın çıxarılması zamanı xəta baş verib : %s"
  },
  "CRM_CUSTOMER_MONGO_DOCUMENT_BY_ID_NOT_FOUND": {
    "en": "CRM customer mongo document with %s id was not found",
    "ru": "Клиент CRM с идентификатором %s не найден",
    "az": "CRM müştərisi %s identifikator ilə tapılmadı"
  },
  "CRM_CUSTOMER_MONGO_DOCUMENT_BY_NAME_EXCEPTION": {
    "en": "CRM customers mongo repository has thrown exception during documents by name extraction : %s",
    "ru": "В репозитории клиентов CRM, во время извлечения по имени произошла ошибка : %s",
    "az": "CRM müştərilər repositoriyda, ad üzrə mətumatın çıxarılması zamanı xəta baş verib : %s"
  },
  "CRM_CUSTOMER_MONGO_DOCUMENT_BY_NAME_NOT_FOUND": {
    "en": "CRM customer mongo document with %s name was not found",
    "ru": "Клиент CRM с именем %s не были найден",
    "az": "CRM müştərisi %s ad ilə tapılmadı"
  },
  "THERE_IS_NO_LISTENER_FOR_ROUTE": {
    "en": "There is no listener for %s route",
    "ru": "Нет слушателя для %s маршрута",
    "az": "%s marşrutu üçün dinləyici yoxdur"
  },
  "METHOD_IS_NOT_SUPPORTED": {
    "en": "%s method is not supported for %s route",
    "ru": "%S метод не поддерживается для %s маршрута",
    "az": "%s metod %s marşrut üçün dəstəklənmir"
  },
  "HTTP_REQUEST_IS_NOT_READABLE": {
    "en": "Request body is not readable for %s route %s method",
    "ru": "Тело запроса не читается для %s маршрута %s http метода",
    "az": "Müraciət oxunacaq formatda döyül %s marşrut üçün və %s http metod üçün"
  },
  "HTTP_REQUEST_FAILED_COMMON_VALIDATION": {
    "en": "Request body did not pass common validation at %s route with %s method",
    "ru": "Тело запроса не прошло общую проверку для %s маршрута %s http метода",
    "az": "Müraciət ümumi yoxlamanı %s marşrut üçün və %s http metod üçün keçmədi"
  },
  "HTTP_REQUEST_FAILED_COMMON_VALIDATION_REASON" : {
    "en": "%s field did not pass validation, reason : %s",
    "ru": "%s поле не прошло валидацию, причина : %s",
    "az": "%s sahə validatisiyanı keçməyib, səbəb : %s"
  },
  "FAILED_ACTUATOR_AUTHENTICATION" : {
    "en": "Request to actuator is not authenticated. Reason: %s",
    "ru": "Запрос в актутор не аутентифицирован. Причина: %s",
    "az": "Aktuatora gedən sorğu autentifikasiyanı keçməyib. Səbəb: %s"
  },
  "CRM_USER_BY_LOGIN_EXTRACTION_NOT_FOUND": {
    "en": "CRM user mysql entity with %s login was not found",
    "ru": "Сущность клиента CRM с логином %s не была найдена",
    "az": "CRM müştərisinin mahiyyəti %s logini ilə tapılmadı"
  },
  "CRM_USER_BY_LOGIN_EXTRACTION_REPOSITORY_EXCEPTION": {
    "en": "CRM users mysql repository has thrown exception during document by login extraction : %s",
    "ru": "В репозитории пользователелей CRM, во время извлечения по логину произошла ошибка : %s",
    "az": "CRM istifadəçilər repositoriyda, login üzrə mətumatın çıxarılması zamanı xəta baş verib : %s"
  },
  "CRM_USER_REPOSITORY_EXCEPTION_FAILED_SAVE": {
    "en": "CRM User repository has thrown exception during entity save : %s",
    "ru": "CRM Пользовательский репозиторий выдал исключение во время сохранения сущности : %s",
    "az": "CRM İstifadəçi repositoriyi mahiyyət yadda saxlanılması zamanı texniki xəta gaytarıb : %s"
  }
}